assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -85,6 +106,10 @@ $filter-btn: '.esprs-button.ant-btn.ee-filter-bar__btn';
 		white-space: pre;
 		width: auto;
 
+		@include max782px {
+			width: 100%;
+		}
```

Comment:
tooltip needs to be excluded here

Reply:
maybe not... it won't do anything on mobile since there is no hover state right?

Reply:
yes, you're right, nothing has been changed, this is how it looks:

![image](https://user-images.githubusercontent.com/1477453/77540955-acc14900-6eac-11ea-9684-4ad8da780ca5.png)

Reply:
without `width: 100%`

![image](https://user-images.githubusercontent.com/1477453/77541032-c5316380-6eac-11ea-96a3-7d9a871b378f.png)

Reply:
that looks much better... think I prefer the 100% width fields for mobile.